readthedocs/core/urls/__init__.py

```diff
@@ -38,6 +38,17 @@
         serve.serve_docs,
         name='docs_detail',
     ),
+    # Serving external version docs
```

Comment:
We don't want to hook this up to `serve_docs`, we can serve it already directly with the media serving code.

Reply:
@ericholscher Yes we can serve it directly through the django media serving but then we won't be able to do this
`https://docs.readthedocs.io/32/` we have to have a file name and extension `https://docs.readthedocs.io/32/index.html`
thats why I did it like this.